### ISSUE TITLE
[FE] My VA: 508-defect-2 [SCREENREADER]: Alert icon in applications in progress cards must have alt text

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25072]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I want to know if something is important or needs my attention if it has a visual warning icon.

## Details

Currently the black exclamation icon does not have any alternative text to provide screen reader users an equivalent experience to sighted users. This can be easily addressed by adding in `<span class="sr-only">Alert: </span>` which I believe is the current method used for these icon fonts.

## Acceptance Criteria

- [ ] Add `<span class="sr-only">Alert: </span>` after the alert icon

## Steps to Recreate

Log in as staging user 10 and inspect the alert icon, confirm there's no alt text on or around it

## Proposed Solution (if known)

Add `<span class="sr-only">Alert: </span>` after the alert icon

## Screenshots or Trace Logs

https://user-images.githubusercontent.com/14154792/119161523-6a9b8500-ba27-11eb-9398-5586b3880c55.mov

**Comment by erikphansen** [comment]
@joshkimux this can be validated on staging

**Comment by joshkimux** [comment]
@erikphansen , validated with user 15! Great work! 🥳
